target/linux/x86/xeon/config-default

```diff
@@ -0,0 +1,262 @@
+CONFIG_64BIT=y
```

Comment:
Instead of introducing a new build target, the kernel configuration changes should be included in ``target/linux/x86/64/config-default``.

The configuration can be done with ``make CONFIG_TARGET=subtarget kernel_menuconfig`` after selecting ``x86_64`` as subtarget.